README.md

```diff
@@ -2,3 +2,9 @@
 A java library for integrating infrastructure components like the jitsi-videobridge, kurento etc. with callstats.io
 
 Bridge Metrics -  http://www.callstats.io/bridge-metrics/
+
+
+VM arguemnts
+
+  -Dlog4j.configurationFile=config/log4j2.xml  #Log4j2 xml configuration file path
```

Comment:
Is this VM argument mandatory? If it is, probably better to highlight this point.